week_1/String/longest_common_prefix.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {string[]} strs
```

Comment:
Let's fill this out with a meaningful bit of documentation to describe what this function is designed to do 👍

Reply:
I added a description to the beginning of the file.